app/static/js/load_weeks.js

```diff
@@ -0,0 +1,43 @@
+document.onload = function () {
+    const container = document.querySelector('.calender-grid');
+}
+
+const anticipating = 1;
+const last = null;
+
+function open_daily(new_days) {
+    let all_days = new_days.getElementsByClassName('day'), i;
+    for (i = 0; i < all_days.length; ++i) {
+        all_days[i].onclick = function () {
+            let daily_event = document.getElementById("day-view");
+            daily_event.classList.toggle("day-view-visible");
+        };
+    }
+    return new_days;
+}
+
+function load_week(last_day) {
+    if (last_day != last) {
```

Comment:
Where is `last` coming from?

Reply:
last is a global parameter set to null and every iteration it gets the last day date display in the board.